frontend/src/.stylelintrc.yml

```diff
@@ -6,18 +6,25 @@ plugins:
   - stylelint-scss
 
 rules:
-  # allow using @use
+  # allow using @use, @each
   at-rule-no-unknown:
     - true
     - ignoreAtRules:
         - use
+        - each
+  property-no-unknown:
+    - true
+    - ignoreSelectors:
+        - :export
   # allow @use to precede @import
   no-invalid-position-at-import-rule:
     - true
     - ignoreAtRules:
         - use
   # allow :global which is specific pseudo class, available in .module.css
+  # allow :export which is specific pseudo class, available in .module.css
```

Comment:
export isn't specific pseudo class for .module.css
It was my comment about ":global" pseudo, which is actually is part of .module.css logic and by default is unknown to css, scss and stylelint.

This is only comment, so you don't really have to change anything, but if you would have a chance, please, fix it :)

Reply:
fixed